tests/wpt/tests/tools/wptrunner/wptrunner/browsers/servodriver.py

```diff
@@ -126,9 +128,34 @@ def make_command(self):
         return [self.webdriver_binary, f"--webdriver={self.port}"] + self.webdriver_args
 
     def cleanup(self):
-        super().cleanup()
         os.remove(self.hosts_path)
 
+    def is_alive(self):
+        if not super().is_alive():
+            return False
+        try:
+            conn = HTTPConnection(self.host, self.port)
+            conn.request("GET", "/status")
+            res = conn.getresponse()
+        except Exception:
+            self.logger.info("Servo has shutted down normally.")
```

Comment:
```suggestion
            self.logger.info("Servo has shut down normally.")
```

Should this perhaps also be `debug`? `info` seems very verbose.

Reply:
Yeah. Do you know how can I make things appear in console, if I use `debug`?